reference/dic_tags.rst

```diff
@@ -470,6 +471,77 @@ To add a new rendering strategy - in addition to the core strategies like
 :class:`Symfony\\Component\\HttpKernel\\Fragment\\FragmentRendererInterface`,
 register it as a service, then tag it with ``kernel.fragment_renderer``.
 
+kernel.locale_aware
+-------------------
+
+.. versionadded:: 4.3
+
+    The ``kernel.locale_aware`` tag was introduced in Symfony 4.3.
+
+**Purpose**: To interact with the locale
```

Comment:
Perhaps add link:ref to what is a locale in symfony request/response context?

Reply:
I would remove this line

Reply:
It doesn't add much, but all tags include a "purpose" line, so let's keep it.